### ISSUE TITLE
Trakt token error, bad request link

### ISSUE THREAD

**Issue by AstroSpiff** [trakt-for-plex/configuration#73]
Good evening, logging into accounts with trakt no longer works, it gives token error. The trakt requests link no longer works. The following link needs to be updated: https://api-v2launch.trakt.tv/oauth/token (no longer active)

**Comment by zluckytraveler** [comment]
Yup same here, can’t get it to work either. Do you know if there is a fork with the fix or something else we can do?

**Comment by Deihmos** [comment]
Did anyone find a solution? It can't be a difficult fix.

**Comment by zluckytraveler** [comment]
> Did anyone find a solution? It can't be a difficult fix.

@Deihmos If you are really determined to resolve this issue and if you do not have any coding skills to do it yourself, you could just hire a [Fiverr](https://www.fiverr.com) worker with coding skills to fork the repo and fix the code.

If you happen to do it, you should definitely share the repo with everyone else or at least me 😆

**Comment by Deihmos** [comment]
It is not that important to me. It was working fine until I decided to do a fresh install not realizing authentication no longer works. I just can't sync for multiple users anymore and that's ok. Maybe one day this will be fixed but I wouldn't hold my breath. It has been almost 10 years since it was last updated.

**Comment by zluckytraveler** [comment]
It’s not just syncing that no longer works. Adding any new users also doesn’t work either. So if by any chance you have to do a fresh reinstall you can’t add anyone back anymore nor add anyone new to your current install.

I’d suggest looking into PlexTraktSync instead, although not as ideal or easy to use for multiple users.

**Comment by Deihmos** [comment]
I was able to get this working. You can either use my fork, fork it yourself or download gh-pages from the source repo. There is just one line that needs editing to api.trakt.tv. 

To get the token you need CORS unblock. Just turn it on when getting the token then you can turn off / remove after. Only needed to get the token. 

http://deihmos.github.io/configuration
https://chromewebstore.google.com/detail/cors-unblock/hadoojkfknbjgoppkecpgamiajljiief

**Comment by karadoulis** [comment]
> I was able to get this working. You can either use my fork, fork it yourself or download gh-pages from the source repo. There is just one line that needs editing to api.trakt.tv.
> 
> To get the token you need CORS unblock. Just turn it on when getting the token then you can turn off / remove after. Only needed to get the token.
> 
> http://deihmos.github.io/configuration https://chromewebstore.google.com/detail/cors-unblock/hadoojkfknbjgoppkecpgamiajljiief

Tried it but it doesn't seem to work either. :/

Unable to find valid connection

**Comment by rg9400** [comment]
> I was able to get this working. You can either use my fork, fork it yourself or download gh-pages from the source repo. There is just one line that needs editing to api.trakt.tv.
> 
> To get the token you need CORS unblock. Just turn it on when getting the token then you can turn off / remove after. Only needed to get the token.
> 
> http://deihmos.github.io/configuration https://chromewebstore.google.com/detail/cors-unblock/hadoojkfknbjgoppkecpgamiajljiief

I'm actually unsure what change you've made. Seems like you've altered a part of the vendor JS?

**Comment by Deihmos** [comment]
> > I was able to get this working. You can either use my fork, fork it yourself or download gh-pages from the source repo. There is just one line that needs editing to api.trakt.tv.
> > To get the token you need CORS unblock. Just turn it on when getting the token then you can turn off / remove after. Only needed to get the token.
> > http://deihmos.github.io/configuration https://chromewebstore.google.com/detail/cors-unblock/hadoojkfknbjgoppkecpgamiajljiief
> 
> I'm actually unsure what change you've made. Seems like you've altered a part of the vendor JS?

All I did was update the link from "https://api-v2launch.trakt.tv" to "https://api.trakt.tv" in file vendor.8d369d42.js. Authentication will only work with CORS unblock. It worked fine for me. You can search for the following in the old code. 

d="https://api-v2launch.trakt.tv",e="1.0.0";return

**Comment by Deihmos** [comment]
I have never had any issues using the trakt code in the app settings for single users. What I did is for the configuring multiple users which can't be done in the app settings on plex.

**Comment by Bruxes** [comment]
Trying @Deihmos url I'm getting "Unable to retrieve token" when trying to log in into Trakt. Using rg9400 fork of Trakt-for-plex.

**Comment by SasSam** [comment]
PR has been raised for the fix.

**Comment by Deihmos** [comment]
The developer hasn't been around for years.

**Comment by SasSam** [comment]
@fuzeman Could you please review the PR? If you are happy, please accept it and release the new version. Thanks in advance!

**Comment by SasSam** [comment]
> The developer hasn't been around for years.

The developer (@fuzeman) was active this year on their other repos. Hopefully they will see this message.

edit: while the PR is pending, I've set up a webpage for this configurator with the fixed version: https://trakt4plex.link/configuration
